docker/sds-api/spine-directory-service/sds/request/cpm.py

```diff
@@ -170,6 +173,68 @@ def validate_filters(self, query_parts):
         if non_empty_count < 2:
             self._raise_invalid_query_params_error()
     
+    def set_mhs_endpoint(self, ldap_results):
+        for key, ldap_result in enumerate(ldap_results):
+            service, interaction = self._extract_service_and_interaction(ldap_result['nhsMhsSvcIA'])
```

Comment:
I'm not sure if this is covered in other parts of the sds fhir api - but the field names coming back from ldap have cases that are all over the place, e.g. `nhsMhsSvcIA` and `nhsmhssvcia` can both be the field name from what i understand

Reply:
Interesting. However this is not actual ldap it's my made up version of ldap after converting the response from CPM